packages/moonstone/VideoPlayer/MediaControls.js

```diff
@@ -154,6 +154,14 @@ const MediaControlsBase = kind({
 		 */
 		moreButtonLabel: PropTypes.string,
 
+		/**
+		 * The method to run when the more button rendered, giving a reference to the DOM.
+		 *
+		 * @type {Function}
+		 * @public
```

Comment:
Should this prop be `public`? If yes, please update CHANGELOG.md `added`.

Reply:
I changed prop to private. Thanks.